projects/angular/src/button/_buttons.clarity.scss

```diff
@@ -5,151 +5,99 @@
   }
 }
 
 @mixin populateButtonProperties($button-type: default) {
-  @include button-css-var($button-type, border-color, border-color, variables.$clr-use-custom-properties);
-  @include button-css-var($button-type, background-color, bg-color, variables.$clr-use-custom-properties);
-  @include button-css-var($button-type, color, color, variables.$clr-use-custom-properties);
+  @include button-css-var($button-type, border-color);
+  @include button-css-var($button-type, background-color, bg-color);
+  @include button-css-var($button-type, color);
 
   cds-icon,
   clr-icon {
-    @include button-css-var($button-type, color, color, variables.$clr-use-custom-properties);
+    @include button-css-var($button-type, color);
   }
 
   &:hover {
-    @include button-css-var($button-type, background-color, hover-bg-color, variables.$clr-use-custom-properties);
-    @include button-css-var($button-type, color, hover-color, variables.$clr-use-custom-properties);
+    @include button-css-var($button-type, background-color, hover-bg-color);
+    @include button-css-var($button-type, color, hover-color);
     cds-icon,
     clr-icon {
-      @include button-css-var($button-type, color, hover-color, variables.$clr-use-custom-properties);
+      @include button-css-var($button-type, color, hover-color);
     }
   }
 
   &:active {
-    @include button-css-var($button-type, background-color, active-bg-color, variables.$clr-use-custom-properties);
-    @include button-css-var($button-type, color, active-color, variables.$clr-use-custom-properties);
+    @include button-css-var($button-type, background-color, active-bg-color);
+    @include button-css-var($button-type, color, active-color);
     cds-icon,
     clr-icon {
-      @include button-css-var($button-type, color, active-color, variables.$clr-use-custom-properties);
+      @include button-css-var($button-type, color, active-color);
     }
   }
 
   &.disabled,
   &:disabled {
-    @include button-css-var($button-type, color, disabled-color, variables.$clr-use-custom-properties);
+    @include button-css-var($button-type, color, disabled-color);
     cursor: not-allowed;
-    @include mixins.css-var(opacity, clr-btn-disabled-opacity, 0.4, variables.$clr-use-custom-properties);
-    @include button-css-var($button-type, background-color, disabled-bg-color, variables.$clr-use-custom-properties);
-    @include button-css-var($button-type, border-color, disabled-border-color, variables.$clr-use-custom-properties);
+    @include button-css-var($button-type, background-color, disabled-bg-color);
+    @include button-css-var($button-type, border-color, disabled-border-color);
+    cds-icon,
+    clr-icon {
+      @include button-css-var($button-type, color, disabled-color);
+    }
   }
 }
 
 @mixin btn-checked-styles($button-type: default, $type: checkbox) {
   // add semantic prop: group-checked-font-color
-  input[type='#{$type}']:checked + label,
```

Comment:
Can you elaborate on the differences here? Is this also refactoring the state declarations from other places in the file to live here?

Reply:
This change is tied to previously missing design specification for checkboxes styled as group buttons. New spec is added in figma.
Old behavior can be found [button group](https://clarity.design/documentation/button-group/code) checkbox section.
The full implementation will be resolved with JIRA CDE-1310.